Islandora/islandora_xml_forms
# Add "not applicable" option to Creative Commons element

**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2173)

# What does this Pull Request do?
Adds "not applicable" options to the Creative Commons form element, which, when selected, cause no URI to be generated. 

Currently, having a CC element in your form forces a URI to be generated. This PR makes the CC URI optional.

# What's new?
Each of the dropdown lists has a "Not applicable" option. If any one of these is selected, the license URI will be blank.

# How should this be tested?

* Create a form with a Creative Commons element in it and use it to ingest an object. Note that any options you select generate a license - no way to say "I don't want one, thanks"
* Check out this branch. Do it again, but this time select the "Not applicable" option for any of the dropdowns. Note that the preview disappears.
* Save/ingest your object. Verify that the Creative Commons element is empty.

# Additional Notes:
At the moment, editing the object with no CC element in the metadata provides the CC option with "yes" already populated. This could probably be solved by setting "Not applicable" as default -- would that be acceptable?

# Interested parties
@Islandora/7-x-1-x-committers
